<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>currentMonitorFirstWindow</class>
 <widget class="QMainWindow" name="currentMonitorFirstWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>currentMonitorFirstWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                            stop:0 #1b2b34,
                            stop:1 #0f2027);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxBluetooth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>431</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                            stop:0 #1b2b34,
                            stop:1 #0f2027);




</string>
    </property>
    <property name="title">
     <string>Bluetooth Connection Panel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="disconnectFromDeviceButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #44627D;
    color: white;
    border-radius: 6px;
    padding: 5px 10px;
}
QPushButton:hover {
    background-color: #5A7D9A;
}</string>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectToDeviceButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #44627D;
    color: white;
    border-radius: 6px;
    padding: 5px 10px;
}
QPushButton:hover {
    background-color: #5A7D9A;
}
</string>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scanBluetoothButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                      stop:0 #90A4AE, stop:1 #607D8B);
    color: white;
    font-weight: bold;
    font-size: 13px;
    border: 1px solid #455A64;
    border-radius: 6px;
    padding: 6px 10px;
}
QPushButton:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                      stop:0 #b0bec5, stop:1 #78909c);
}
QPushButton:pressed {
    background-color: #455A64;
}
</string>
     </property>
     <property name="text">
      <string>SCAN</string>
     </property>
    </widget>
    <widget class="QComboBox" name="devicesComboBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>261</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: qlineargradient(
    x1:0, y1:0, x2:1, y2:1,
    stop:0 #0f2027,
    stop:0.5 #203a43,
    stop:1 #2c5364
);</string>
     </property>
    </widget>
    <widget class="QLabel" name="bluetoothStatusLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #CCCCCC;
    font-size: 12px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxMotor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>431</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                            stop:0 #1b2b34,
                            stop:1 #0f2027);


QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 5px;
    font-weight: bold;
    font-size: 14px;
    color: #dddddd;
}</string>
    </property>
    <property name="title">
     <string>Motor Control Panel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <widget class="QSlider" name="pwmSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>160</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #444;
    height: 8px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                stop:0 #2c3e50, stop:1 #3498db);
    border-radius: 4px;
}

QSlider::handle:horizontal {
    background: white;
    border: 1px solid #1e88e5;
    width: 16px;
    height: 16px;
    margin: -5px 0;
    border-radius: 8px;
}

QSlider::add-page:horizontal {
    background: #ccc;
}

QSlider::sub-page:horizontal {
    background: #3498db;
    border-radius: 4px;
}
</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="pwmSpinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #3e3e3e, stop:1 #2c2c2c);
    color: white;
    border: 1px solid #555;
    border-radius: 6px;
    padding: 4px 8px;
    font-weight: bold;
    font-size: 14px;
}

QSpinBox::up-button {
    subcontrol-origin: border;
    subcontrol-position: top right;
    width: 16px;
    background-color: #4CAF50;
    border-left: 1px solid #333;
    border-bottom: 1px solid #333;
    border-top-right-radius: 6px;
}

QSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position: bottom right;
    width: 16px;
    background-color: #EF5350;
    border-left: 1px solid #333;
    border-top: 1px solid #333;
    border-bottom-right-radius: 6px;
}

QSpinBox::up-arrow {
    image: url(:/icons/up-arrow.png);  /* varsayılan Qt okları da olur */
    width: 10px;
    height: 10px;
}

QSpinBox::down-arrow {
    image: url(:/icons/down-arrow.png);
    width: 10px;
    height: 10px;
}
</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
    <widget class="QPushButton" name="reverseButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #ff9900, stop:1 #ff6600);
    color: white;
    font-weight: bold;
    border: 1px solid #cc6600;
    border-radius: 6px;
    padding: 6px 12px;
}
QPushButton:hover {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #ffaa33, stop:1 #ff8800);
}
QPushButton:pressed {
    background-color: #b35900;
}
</string>
     </property>
     <property name="text">
      <string>Reverse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="forwardButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                      stop:0 #4CAF50, stop:1 #388E3C);
    color: white;
    font-weight: bold;
    font-size: 14px;
    border: 1px solid #2e7d32;
    border-radius: 6px;
    padding: 6px 12px;
}
QPushButton:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                      stop:0 #66bb6a, stop:1 #43a047);
}
QPushButton:pressed {
    background-color: #2e7d32;
}
</string>
     </property>
     <property name="text">
      <string>Forward</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                      stop:0 #EF5350, stop:1 #C62828);
    color: white;
    font-weight: bold;
    font-size: 14px;
    border: 1px solid #b71c1c;
    border-radius: 6px;
    padding: 6px 12px;
}
QPushButton:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                      stop:0 #e57373, stop:1 #d32f2f);
}
QPushButton:pressed {
    background-color: #b71c1c;
}
</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="motorStatusLabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">ui-&gt;motorStatusLabel-&gt;setStyleSheet(
    &quot;background-color: rgba(0, 0, 0, 30);&quot;
    &quot;border-radius: 5px;&quot;
    &quot;padding: 4px 8px;&quot;
    &quot;font-weight: bold;&quot;
    &quot;color: green;&quot;
);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="directionLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Direction           </string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendCommandsButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                      stop:0 #42A5F5, stop:1 #1E88E5);
    color: white;
    font-weight: bold;
    font-size: 14px;
    border: 1px solid #1565c0;
    border-radius: 6px;
    padding: 6px 12px;
}
QPushButton:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                      stop:0 #64b5f6, stop:1 #2196f3);
}
QPushButton:pressed {
    background-color: #1565c0;
}
</string>
     </property>
     <property name="text">
      <string>Send Commands</string>
     </property>
    </widget>
    <widget class="QLabel" name="pwmValueLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PWM</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMinMax">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Min: -- | Max: --</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>751</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                            stop:0 #1b2b34,
                            stop:1 #0f2027);



</string>
    </property>
    <property name="title">
     <string>MQTT Connection Panel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <widget class="QLineEdit" name="hostLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="portLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="publishLineEdit">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>160</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="connectMQTTButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #44627D;
    color: white;
    border-radius: 6px;
    padding: 5px 10px;
}
QPushButton:hover {
    background-color: #5A7D9A;
}</string>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnectMQTTButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #44627D;
    color: white;
    border-radius: 6px;
    padding: 5px 10px;
}
QPushButton:hover {
    background-color: #5A7D9A;
}</string>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="publishButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>200</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
        background-color: #2e86de;
        color: white;
        font-weight: bold;
        border: 1px solid #1e5faf;
        border-radius: 6px;
        padding: 5px 12px;
    }
    QPushButton:hover {
        background-color: #54a0ff;
    }
    QPushButton:pressed {
        background-color: #1e5faf;
    }
    QPushButton:disabled {
        background-color: #3b3b3b;
        color: #aaaaaa;
    }</string>
     </property>
     <property name="text">
      <string>Publish</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="receivedTextEdit">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>280</y>
       <width>301</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="txLabel">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Transmitted Messages</string>
     </property>
    </widget>
    <widget class="QLabel" name="rxLabel">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>250</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Received Messages</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="clientID">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="topicPublish">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>200</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">ui-&gt;setPlaceholderText(&quot;Enter topic name (e.g. motor/current)&quot;);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">    QLabel {
        color: #dcdde1;
        font-size: 12px;
        font-weight: 500;
        letter-spacing: 0.5px;
    }</string>
     </property>
     <property name="text">
      <string>HOST</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">    QLabel {
        color: #dcdde1;
        font-size: 12px;
        font-weight: 500;
        letter-spacing: 0.5px;
    }</string>
     </property>
     <property name="text">
      <string>PORT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">    QLabel {
        color: #dcdde1;
        font-size: 12px;
        font-weight: 500;
        letter-spacing: 0.5px;
    }</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">    QLabel {
        color: #dcdde1;
        font-size: 12px;
        font-weight: 500;
        letter-spacing: 0.5px;
    }</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">    QLabel {
        color: #dcdde1;
        font-size: 12px;
        font-weight: 500;
        letter-spacing: 0.5px;
    }</string>
     </property>
     <property name="text">
      <string>Client ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="topicSubscribe">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>370</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="subscribeButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>370</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
        background-color: #2e86de;
        color: white;
        font-weight: bold;
        border: 1px solid #1e5faf;
        border-radius: 6px;
        padding: 5px 12px;
    }
    QPushButton:hover {
        background-color: #54a0ff;
    }
    QPushButton:pressed {
        background-color: #1e5faf;
    }
    QPushButton:disabled {
        background-color: #3b3b3b;
        color: #aaaaaa;
    }</string>
     </property>
     <property name="text">
      <string>Subscribe</string>
     </property>
    </widget>
    <widget class="QLabel" name="mqttStatusLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="transmittedPublishTextEdit">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>301</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>410</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
        background-color: #2e86de;
        color: white;
        font-weight: bold;
        border: 1px solid #1e5faf;
        border-radius: 6px;
        padding: 5px 12px;
    }
    QPushButton:hover {
        background-color: #54a0ff;
    }
    QPushButton:pressed {
        background-color: #1e5faf;
    }
    QPushButton:disabled {
        background-color: #3b3b3b;
        color: #aaaaaa;
    }</string>
     </property>
     <property name="text">
      <string>Unsubscribe</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>431</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="chartLayout">
     <item>
      <widget class="QChartView" name="chartWidget" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxLog">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>480</y>
      <width>751</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                            stop:0 #1b2b34,
                            stop:1 #0f2027);


QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 6px;
    color: #ffffff;
    font-weight: bold;
    font-size: 14px;
    background-color: transparent;
}</string>
    </property>
    <property name="title">
     <string>Motor and Bluetooth Activity</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <widget class="QTextEdit" name="bluetoothTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>341</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="alarmTextEdit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>341</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: gray; font-style: italic</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:italic;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-style:normal;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1215</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
